File: app/src/main/java/com/task/paginationlist/presentation/piclist/views/fragments/intefaces/IPictureView.java
package com.task.paginationlist.presentation.piclist.views.fragments.intefaces;

import com.arellomobile.mvp.MvpView;
import com.arellomobile.mvp.viewstate.strategy.SkipStrategy;
import com.arellomobile.mvp.viewstate.strategy.StateStrategyType;


public interface IPictureView extends MvpView {
    @StateStrategyType(SkipStrategy.class)
    void setImageUrl(String url);


}

File: app/src/main/java/com/task/paginationlist/presentation/piclist/views/fragments/intefaces/IPictureView.kt
package com.task.paginationlist.presentation.piclist.views.fragments.intefaces

import com.arellomobile.mvp.MvpView
import com.arellomobile.mvp.viewstate.strategy.SkipStrategy
import com.arellomobile.mvp.viewstate.strategy.StateStrategyType


interface IPictureView : MvpView {
    @StateStrategyType(SkipStrategy::class)
    fun setImageUrl(url: String)


}
